.github/workflows/create-release.yml

```diff
@@ -131,27 +115,196 @@ jobs:
           done
           docker image inspect quay.io/${{ matrix.projects.image }}:${{ inputs.version }}
 
+  test:
+    name: Test Konveyor
+    needs: release-components
+    uses: konveyor/ci/.github/workflows/global-ci-bundle.yml@main
+    with:
+      operator: quay.io/konveyor/tackle2-operator:${{ inputs.version }}
+      tackle_hub: quay.io/konveyor/tackle2-hub:${{ inputs.version }}
+      keycloack_init: quay.io/konveyor/tackle-keycloak-init:${{ inputs.version }}
+      tackle_ui: quay.io/konveyor/tackle2-ui:${{ inputs.version }}
+      addon_admin: quay.io/konveyor/tackle2-addon:${{ inputs.version }}
+      addon_analyzer: quay.io/konveyor/tackle2-addon-analyzer:${{ inputs.version }}
+
   publish:
     name: Build and Push Manifest
-    needs: release-components
```

Comment:
do we want this one to depend on tests or something?

Reply:
it should. Good find.